components/dash-core-components/src/components/RangeSlider.react.js

```diff
@@ -125,6 +125,29 @@ RangeSlider.propTypes = {
             'bottomLeft',
             'bottomRight',
         ]),
+        /**
+         * Format to apply to the tooltip
+         * The string must contain `{value}`
+         */
+        format: PropTypes.string,
+        /**
+         * Custom style for the tooltip.
+         */
+        style: PropTypes.object,
+        /**
+         * Reference to a function in the `window.dccFunctions` namespace.
+         * This can be added in a scrip in the asset folder.
+         *
+         * For example, in `assets/tooltip.js`:
+         * ```
+         * window.dccFunctions = window.dccFunctions || {};
+         * window.dccFunctions.multByTen = function(value) {
+         *     return value * 10;
+         * }
+         * ```
+         * Then in the component `tooltip={'transform': 'multByTen'
```

Comment:
```suggestion
         * Then in the component `tooltip={'transform': 'multByTen'}
```

Is there any way to call this "function" instead of "transform" so it's consistent with dash-ag-grid?

Reply:
```suggestion
         * Then in the component `tooltip={'transform': 'multByTen'}`
```
Also wants a trailing backtick ;)

I'd be ok with calling this `function`, though for dash-ag-grid we need a generic word since it gets used in a lot of different contexts, whereas here it's specifically transforming the data value into the display value.

Like in `format`/`template` above, we should say something about how these two interact, perhaps "If both `template` and `function` are used, the result of `function` is inserted into the `template`. Alternatively you can just use `function` to construct the final string to display".

Reply:
It would be consistent with this pattern in AG Grid - providing a function that returns a formatted string.
```
 "valueFormatter": {"function": "d3.format('($,.2f')(params.value)"},

```
In fact, if you were already using AG Grid in the app, could these d3 functions be available to the tooltip as well?  That would be pretty cool

Reply:
We could if we brought dash-ag-grid into the core, otherwise we wouldn’t want to couple these two packages together like that

Reply:
Actually, the implementation here is quite different from that in `dash-ag-grid`, in that all we accept here is the name of the function to look for in `window.dccFunctions`, not a whole expression to be parsed. I think this makes sense, because these are much simpler functions than we need to support in `dash-ag-grid` where there's this huge parameter object you may need to access, and it's better to keep dcc simpler and lighter. So it might even be good to not try for naming consistency as that could confuse people into thinking they have feature parity.